base/gmp.jl

```diff
@@ -547,6 +547,10 @@ end
 (-)(x::BigInt) = MPZ.neg(x)
 (~)(x::BigInt) = MPZ.com(x)
 
+# nand and nor
+nand(a::BigInt, b::BigInt) = MPZ.com(MPZ.and(a, b))
+nor(x::BigInt, y::BigInt) = MPZ.com(MPZ.ior(x, y))
```

Comment:
Why are these definitions needed?  Aren't they equivalent to the generic `~(x & y)` and `~(x | y)` fallback definitions?

Reply:
You're right, they are equivalent. I will remove them.